charts/internal/shoot-storageclasses/values.yaml

```
@@ -1 +1,2 @@
 useLegacyProvisioner: true
+encrypt: false
```

```suggestion
encrypted: false
```

to be consistent with the other value names